addons/wp/stringtable.xml

```diff
@@ -1221,5 +1221,59 @@
             <Italian>Registri artiglieria per uso dinamico</Italian>
             <Chinesesimp>动态使用方面的火炮指示</Chinesesimp>
         </Key>
+        <Key ID="STR_Lambs_WP_Waypoint_Deprecated_TaskAssault_displayName">
+            <English>TASK ASSAULT</English>
+        </Key>
+        <Key ID="STR_Lambs_WP_Waypoint_Deprecated_TaskAssault_Tooltip">
+            <English>Rushes to target position with no regard for safety\nDEPRECATED, USE MODULES FOR MORE FLEXIBILITY FOR MORE FLEXIBILITY</English>
```

Comment:
`USE MODULES FOR MORE FLEXIBILITY FOR MORE FLEXIBILITY` - looks like a duplicate snuck in there.

Reply:
i blame @nk3nny